src/main/java/org/gridsuite/study/server/dto/modification/ModificationType.java

```diff
@@ -7,15 +7,18 @@
 package org.gridsuite.study.server.dto.modification;
 
 import com.powsybl.commons.PowsyblException;
+import io.swagger.v3.oas.annotations.media.Schema;
 import org.gridsuite.study.server.StudyException;
 
 /**
  * @author Slimane Amar <slimane.amar at rte-france.com>
  */
+@Schema(type = "string", allowableValues = { "generators-modification" })
```

Comment:
this "allowableValues" sounds weird, why do we need it only for "generators-modification" ?

Reply:
because it's an enum ;), and for the moment i only allow for one, but in the future, more modification will be allowed but GENERATOR_MODIFICATION in url is not really eye friendly